Resources/config/admin.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <parameters>
+        <!-- ARTICLE -->
+        <parameter key="sonata.article.admin.article.class">Sonata\ArticleBundle\Admin\ArticleAdmin</parameter>
```

Comment:
AFAIK this is not recommended for bundle. This should be done trough config file if needed.